scripts/onsite/onsite_create_calibration_file.py

```diff
@@ -0,0 +1,120 @@
+#!/usr//bin/env python
+
+"""
+ F. Cassol, 12/11/2019
+
+ Onsite script for creating a flat-field calibration file file to be run as a command line:
+
+ --> onsite_create_calibration_file
+
+"""
+
+import argparse
+from pathlib import Path
+from lstchain.io.data_management import *
+import lstchain.visualization.plot_calib as calib
+
+# to be changed with the right one : /fefs/aswg/data/real
+base_dir = '/ctadata/franca/fefs/aswg/real'
+#base_dir = '/fefs/aswg/data/real'
+
+# parse arguments
+parser = argparse.ArgumentParser(description='Create flat-field calibration files',
+                                 formatter_class = argparse.ArgumentDefaultsHelpFormatter)
+required = parser.add_argument_group('required arguments')
+optional = parser.add_argument_group('optional arguments')
+
+required.add_argument('-r', '--run_number', help="Run number if the flat-field data",
+                      type=int, required=True)
+required.add_argument('-p', '--pedestal_run', help="Run number of the drs4 pedestal run",
+                      type=int, required=True)
+
+optional.add_argument('-v', '--version', help="Version of the production",
+                      type=int, default=0)
+optional.add_argument('-s', '--statistics', help="Number of events for the flat-field and pedestal statistics",
+                      type=int, default=10000)
+
+args = parser.parse_args()
+run = args.run_number
+ped_run = args.pedestal_run
+prod_id = 'v%02d'%args.version
+stat_events = args.statistics
+max_events = 1000000
```

Comment:
I have problem to pass None, anyway onsite this number is not supposed to change, when I have time I try to understand the problem with the None

Reply:
https://stackoverflow.com/questions/48295246/how-to-pass-none-keyword-as-command-line-argument

Reply:
wait, don't you want to have `max_events = stat_events`?
I don't understand the difference here since you state in doc "Number of events for the flat-field and pedestal statistics"

Reply:
max_event is the number of event to be read, stat_events are the one used for estimating the pedestals and flat-field parameters (mean, std, etc...). I can have then have more calibration events each corresponding to a stat_event statistics.

Reply:
So in any case you want `max_events > stat_events` right?

Reply:
I agree with @rlopezcoto, if 1000000 is just an arbitrarily large number, use None as default (it's not an input parameter anyway).

Reply:
> So in any case you want `max_events > stat_events` right?

yes, but it is not a problem if max_events is less, the script will simply stop at the end of the file and collect a max_event statistics. Anyway, onsite the user is not supposed to change the default values. He has simply to give to run numbers